netket/graph/abstract_graph.py

```diff
@@ -1,17 +1,14 @@
 import abc
-import numpy as _np
 
 
 class AbstractGraph(abc.ABC):
     """Abstract class for NetKet graph objects"""
 
-    @property
     @abc.abstractmethod
     def is_connected(self):
         r"""bool: True if the graph is connected"""
         raise NotImplementedError
 
-    @property
```

Comment:
why are those no longer properties?

Reply:
mostly for consistency with what happens internally in networkx: basically knowing if it is connected involves doing some operations, it's not a static attribute. Same for automorphisms etc